examples/auction/src/lib.rs

```diff
@@ -219,10 +212,10 @@ mod tests {
         assert_eq!(actual, err);
```

Comment:
I know you did not add this, but would you please change this to use `claim_eq!` instead, so the test [fails properly when targeting wasm](https://developer.concordium.software/en/mainnet/smart-contracts/tutorials/piggy-bank/testing.html#compiling-and-running-tests-in-wasm).
And the above line to use [`expect_err_report`](https://docs.rs/concordium-std/latest/concordium_std/trait.ExpectErrReport.html#tymethod.expect_err_report) for the same reason